special_edition/asteroid_city.yaml

```diff
@@ -0,0 +1,33 @@
+accent: '#D83E1C'
+background: '#c4f4f1'
+details: darker
+foreground: '#22211a'
+
+############################################################### SEE BELOW
+background_image:
+  # the path is relative to ~/.warp/themes/
+  # the full path to the picture is: ~/.warp/themes/warp.jpg
+  path: ~/.warp/themes/asteroid_city/asteroid_city_wallpaper.jpg
```

Comment:
You wanna use a relative path not the absolute:

`special_edition/asteroid_city_wallpaper.jpg'

Reply:
Done! Made the change. It doesn't work for me locally, but I'm assuming it'll work as expected when this image gets included in the system folder?

Reply:
I'll try to debug this and see what's up

Reply:
I just tried this locally and it seems to work alright for me! Not sure why this doesn't work for Melanie.

<img width="1405" alt="image" src="https://github.com/warpdotdev/themes/assets/24748963/dba0b4e6-ffcf-4200-8c50-78df485e8ee0">